server/handlers/likeHandler.ts

```diff
@@ -0,0 +1,43 @@
+import { CreateLikeRequest, CreateLikeResponse, GetLikesRequest, GetLikesResponse,  } from '../api';
+import { db } from '../datastore';
+import { ExpressHandler, Like } from '../types';
+
+export const createLikeHandler: ExpressHandler<CreateLikeRequest, CreateLikeResponse> = (
+  req,
+  res
+) => {
+  if (!req.body.postId) {
+    return res.sendStatus(400).send('No Post Id');
+  }
+
+  if (!req.body.userId) {
+    return res.sendStatus(400).send('No User Id');
+  }
+
+  let found = db.isDuplicateLike({
```

Comment:
nitpick: Use `const` rather then `let` whenever possible, this makes the code a little more predictable and produces fewer surprises. Same below on L39.